1.8/administration/tls-ssl/haproxy-adminrouter.md

```diff
@@ -1,10 +1,10 @@
 ---
 post_title: Configuring HAProxy in Front of Admin Router
 nav_title: HAProxy in Front of Admin Router
-menu_order: 3.3
+menu_order: 6
 ---
 
-You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/docs/1.8/overview/concepts/#adminrouter).
+You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/docs/1.8/overview/concepts/#adminrouter). This can be useful if you want to put SSL with your own certificate on top of DC/OS. DC/OS does not currently support adding your own certificates directly into Admin Router.
```

Comment:
I prefer "TLS" over "SSL", but this here actually is about HTTPS (HTTP over TLS). Also, "own certificate on top of DC/OS" is technically too vague.

I think this is better:

 "For example, this can be useful if you want to present a custom server certificate to user agents connecting to the cluster via HTTPS."